v3/src/components/case-table/use-columns.tsx

```diff
@@ -109,8 +111,10 @@ export const useColumns = ({ data, indexColumn }: IUseColumnsProps) => {
                 // which leads to undesirable browser behavior.
                 width: kDefaultColumnWidth,
                 resizable: true,
-                headerCellClass: "codap-column-header",
-                renderHeaderCell: ColumnHeader,
+                headerCellClass: `codap-column-header ${attrIdToEdit === id ? "selected" : ""}`,
+                renderHeaderCell:
+                  (props) =>
+                    <ColumnHeader {...props} attrIdToEdit={attrIdToEdit} setAttrIdToEdit={setAttrIdToEdit} />,
```

Comment:
There's a potential performance issue here in that changes to the column definitions require re-rendering the entire table. If `attrIdToEdit` were managed by the `CollectionTableModel`, then `ColumnHeader` could access them directly and no change would be required here. If the `selected` class could be applied at the `ColumnHeader` level that would be preferable as well, but I'm not sure if that's a straightforward change.